tests/radshock_dimensionless.in

```diff
@@ -13,7 +13,7 @@ amr.v              = 0       # verbosity in Amr
 # *****************************************************************
 # Resolution and refinement
 # *****************************************************************
-amr.n_cell          = 512 4 4
+amr.n_cell          = 256 4 4
 amr.max_level       = 0     # number of levels = max_level + 1
 amr.blocking_factor = 4     # grid size must be divisible by this
```

Comment:
Same comment as above -- setting `amr.max_grid_size_x = 256` should make this much faster on GPU.